app/src/main/java/ar/com/wolox/android/example/model/News.kt

```diff
@@ -13,8 +14,11 @@ data class News(
     var createdAt: String,
     var title: String,
     var picture: String,
-    var text: String
-) {
+    var text: String,
+    @SerializedName("likes")
+    val likes: List<Int>? = null,
```

Comment:
Move this to line above, so I can understand that it makes reference to serialized likes